libcloudforensics/providers/gcp/internal/bigquery.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google BigQuery functionalities."""
+
+from typing import TYPE_CHECKING, List, Dict, Any, Optional
+from libcloudforensics.providers.gcp.internal import common
+
+if TYPE_CHECKING:
+  import googleapiclient.discovery
+
+
+class GoogleBigQuery:
+  """Class to call Google BigQuery APIs.
+
+  Attributes:
+    project_id: Google Cloud project ID.
+  """
+  BIGQUERY_API_VERSION = 'v2'
+
+  def __init__(self, project_id: Optional[str] = None) -> None:
+    """Initialize the GoogleBigQuery object.
+
+    Args:
+      project_id (str): Optional. Google Cloud project ID.
```

Comment:
adding type to docs is obsolete if you already use type annotations, you can remove these from all docs and add them only to class attributes